packages/opencensus-exporter-stackdriver/src/types.ts

```diff
@@ -42,6 +42,8 @@ export interface StackdriverExporterOptions extends ExporterConfig {
    * projectId project id defined to stackdriver
    */
   projectId: string;
+  /** Prefix for metric */
```

Comment:
Add a note on when the user might want to set this, and when they don't need to.

Reply:
Done